### ISSUE TITLE
Random Sample算子

### ISSUE THREAD

**Issue by PanZezhong1725** [PanZezhong1725/operators#22]
randomSample(Tensor[uint32] indices, Tensor logits, float top_p, uint32 top_k, float temperature)

以logits的最后一维采样，将采样的序号放入indices对应位置。

- [x] CPU
- [x] CUDA
- [x] BANG
